src/BinSkim.Driver/MultithreadedAnalyzeCommand.cs

```diff
@@ -69,6 +69,24 @@ public override BinaryAnalyzerContext InitializeGlobalContextFromOptions(Analyze
 
             // Update context object based on command-line parameters.
             context.SymbolPath = options.SymbolsPath;
+
+            IList<string> inaccessibleSymbolPaths = DriverUtilities.GetInaccessibleSymbolPaths(context.SymbolPath);
+
+            if (inaccessibleSymbolPaths.Count > 0)
+            {
+                // These symbol servers provided are inaccessible: {0}.
```

Comment:
> // These symbol servers provided are inaccessible: {0}. [](http://example.com/codeflow?start=15&length=56)

The following symbol servers were specified via the --sympath argument but can't be reached:

Reply:
One or more servers specified in the --sympath argument could not be reached ([https://symweb](https://symweb), [https://blah](https://blah)).